service/src/main/java/bio/terra/cbas/controllers/PublicApiController.java

```diff
@@ -1,16 +1,43 @@
 package bio.terra.cbas.controllers;
 
 import bio.terra.cbas.api.PublicApi;
+import bio.terra.cbas.config.CromwellServerConfiguration;
 import bio.terra.cbas.model.SystemStatus;
+import bio.terra.cbas.model.SystemStatusSystems;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 import org.springframework.stereotype.Controller;
+import org.springframework.web.client.RestTemplate;
 
 @Controller
 public class PublicApiController implements PublicApi {
 
+  private final CromwellServerConfiguration cromwellConfig;
+
+  @Autowired
+  public PublicApiController(CromwellServerConfiguration cromwellConfig) {
+    this.cromwellConfig = cromwellConfig;
+  }
+
   @Override
   public ResponseEntity<SystemStatus> getStatus() {
-    return new ResponseEntity<>(new SystemStatus().ok(true), HttpStatus.OK);
+
+    RestTemplate restTemplate = new RestTemplate();
+    String result;
+    Boolean isOk;
+    try {
+      result = restTemplate.getForObject(this.cromwellConfig.healthUri(), String.class);
+      isOk = true;
+    } catch (Exception e) {
```

Comment:
I've never used `RestTemplate` and this looks OK but there is a JDK builtin for reading from URLs, and I usually prefer JDK APIs when possible.
```suggestion
    try (var stream = new URL(cromwellConfig.healthUri()).openStream()) {
      result = new String(stream.readAllBytes());
      isOk = true;
    } catch (Exception e) {
```

Reply:
Thanks, I did not implement this exact suggestion, but I switched to a different JDK built-in that gave me the hook to add a timeout (from debugging various Cromwell IO things I have a paranoia of infinite timeouts)